next/components/forms/segments/AccountSections/MyApplicationsSection/MyApplicationsHeader.tsx

```diff
@@ -31,17 +51,27 @@ const MyApplicationsHeader = (props: MyApplicationsHeaderBase) => {
           {headerNavigationList.map((item, i) => (
             <li className="w-full lg:w-max" key={i}>
               <button
+                onClick={() => {
+                  router
+                    .push(
+                      {
+                        pathname: router.pathname,
+                        query: { ...router.query, sekcia: englishToSlovakSectionNames[item.tag] },
+                      },
+                      undefined,
+                      { shallow: true },
```

Comment:
note, once pagination is moved into query params as well, if there are no other blockers to such approach, we may even move loading the appropriate data to server side, making this shallow: false, and controlling the entire pagination though server-side-props

take this just as an idea worthy of exploring, no actionable now